Proposed update to README.md

davidbayley80 opened the pull request:
- An example HTML doc would make this much easier for noobs (like me!) to get the code up and running in a standalone browser. 
- Added 'league-container' as requested. 
- Modified the 'renderLeague' example. As you only added 4 teams in the previous 'addTeam' example,  you can't render the league table without either adding more teams or modifying the promotion/relegation positions. I've therefore changed the dropdown to 3, and the promotion and relegation zones to 1 and 4, respectively.

davidbayley80 committed:
Proposed update to README.md

- An example HTML doc would make this much easier for noobs (like me!) to get the code up and running in a standalone browser. 
- Added 'league-container' as requested. 
- Modified the 'renderLeague' example. As you only added 4 teams in the previous 'addTeam' example,  you can't render the league table without either adding more teams or modifying the promotion/relegation positions. I've therefore changed the dropdown to 3, and the promotion and relegation zones to 1 and 4, respectively.

jrobind commented on README.md:
````diff
@@ -17,6 +17,8 @@ Load the files straight into your HTML from the ```dist``` folder:
 <link rel="stylesheet" href="soccer.min.css">
 <script src="soccer.min.js"></script>
 ```
+An example html document is included in the distribution as an example. 
````
Do you mean within `/docs`?

davidbayley80 replied:
Yes, a HTML containing the relevant markup and classes.

davidbayley80 replied:
Happy to pick this up btw^

jrobind replied:
So, at the moment within `/docs` there is already an html file which is for https://jrobind.github.io/soccer.js/ (`/docs` is used for GitHub pages). I'm not sure how much additional value adding another file will bring. Maybe creating a UI to interact with the library would be a cool thing to do? Let me know if you'd be happy to take this on!

jrobind replied:
If you remove this comment and also the comment regarding the unnecessary wrapping div below, I can merge this!

davidbayley80 replied:
> So, at the moment within `/docs` there is already an html file which is for https://jrobind.github.io/soccer.js/ (`/docs` is used for GitHub pages). I'm not sure how much additional value adding another file will bring. Maybe creating a UI to interact with the library would be a cool thing to do? Let me know if you'd be happy to take this on!

I actually didn't realise that the HTML within the /docs was interactive lol. Agreed, no need for a further example :)

davidbayley80 replied:
> If you remove this comment and also the comment regarding the unnecessary wrapping div below, I can merge this!

That should be done now (slowly learning how to use GitHub properly!)

davidbayley80 replied:
I'd be happy to take up adding a UI functionality for the app. Sounds like a good first js project to get my teeth stuck into. 

Would you recommend glitch as an IDE?

jrobind replied:
Glitch is cool, totally up to you! I use a text editor called VS Code which is pretty popular with frontend devs.

jrobind commented on README.md:
````diff
@@ -27,7 +29,9 @@ require(['soccer'], function (soccer) {
 
 ```html
-<div class="league-table"></div>
+  <div class="league-container">
````
I've just checked this again, we actually only need `<div class="league-table"></div>` and not the wrapping container div.

jrobind reviewed (commented):
Thanks for the pull request! Helping to improve the documentation is super useful and much appreciated.

Looks like we no longer need the wrapping container div, but good spot regarding the zones and dropdown config within `renderLeague()` – the proposed updates look great! 

I've left a couple of comments, if you can review them and make a few changes I can approve and merge.

Thanks again!

davidbayley80 committed:
Update README.md

- Removed request for HTML template example 
- Removed unnecessary container: <div class="league-container">

davidbayley80 committed:
Update README.md

jrobind reviewed (approved):
Looks great, thanks for the updates!
